modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/exportimport/content/processor/DDMFormValuesExportImportContentProcessor.java

```diff
@@ -689,6 +690,13 @@ public void transform(DDMFormFieldValue ddmFormFieldValue)
 
 					if (className.equals(Layout.class.getName())) {
 						String uuid = element.attributeValue("uuid");
+
+						if (jsonObject.has("id") &&
+							!Objects.equals(uuid, jsonObject.getString("id"))) {
+
+							continue;
+						}
```

Comment:
The pull looks good to me! :slightly_smiling_face: The only thing that seems a little inconsistent is the JSON Object store the `uuid` as `id` instead of `uuid`, although it was a decision made when the field Link to Page Field was created by Echo Team. Therefore, I'll forward this pull right now.

Thanks